Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient+Customers.h

```diff
@@ -36,7 +37,7 @@
  *  @param customer A BUYCustomer
  *  @param error    An optional NSError
  */
-typedef void (^BUYDataCustomerBlock)(BUYCustomer *customer, NSError *error);
+typedef void (^BUYDataCustomerBlock)(BUYCustomer * _Nullable customer, NSError * _Nullable error);
```

Comment:
As far as I know, there is no `NS_ASSUME_NULLABLE` or similar macro . Just `NS_ASSUME_NONNULL_BEGIN` and `NS_ASSUME_NONNULL_END`.